### DISCUSSION TITLE
Is there a way to read a field for a data label?

### DISCUSSION THREAD

**Discussion by simonwong7** [kurkle/chartjs-chart-sankey#69] [General]
Much in the same way that we can use a function for colorFrom and colorTo, is there a way to get a custom label?

labels: (c) => this.getColor(c.dataset.data[c.dataIndex].someCustomField)

Thanks!

**Comment by kurkle** [comment]
No, not currently possible. This would not be hard to implement though.

**Comment by RishikeshB** [comment]
can u tell me how to set background color for labels has shown in below example:
![labels](https://github.com/kurkle/chartjs-chart-sankey/assets/94789179/499506e9-c33a-438d-8fd8-4f0e8b15ccc8)
